kroxylicious-test-tools/pom.xml

```diff
@@ -86,6 +89,11 @@
             <groupId>org.hamcrest</groupId>
             <artifactId>hamcrest</artifactId>
         </dependency>
+        <dependency>
+            <groupId>info.schnatterer.moby-names-generator</groupId>
+            <artifactId>moby-names-generator</artifactId>
+            <version>${moby-names-generator.version}</version>
+        </dependency>
```

Comment:
I thought it would be nice to generate human readable topic names rather than UUID's to make reading logs easier.